src/main.rs

```diff
@@ -333,14 +332,14 @@ async fn find_city(conf: Conf<'_>) -> Result<(), ()> {
         let name: String = row.get("name");
         let country: String = row.get("country");
         let state: String = row.get("state");
-        if state == "" {
-            text += &format!("{}. {},{}\n", i, name, country);
+        if state.is_empty() {
+            writeln!(&mut text, "{}. {},{}", i, name, country).unwrap();
         } else {
-            text += &format!("{}. {},{},{}\n", i, name, country, state);
+            writeln!(&mut text, "{}. {},{},{}", i, name, country, state).unwrap();
```

Comment:
Some future here we need to include in BotError this error

Reply:
dont like this unwrap , temporal solution

Reply:
Solve in [362c54d](https://github.com/pxp9/weather_bot_rust/pull/6/commits/362c54d3e9463ae2dbf3841e317230c41a0fa131)